include/partition/graph_view.hpp

```diff
@@ -49,7 +49,7 @@ class EdgeIDIterator : public boost::iterator_facade<EdgeIDIterator,
     void decrement() { --position; }
     void advance(difference_type offset) { position += offset; }
     bool equal(const EdgeIDIterator &other) const { return position == other.position; }
-    const reference dereference() const { return position; }
+    reference dereference() const { return position; }
```

Comment:
since `reference` is already `const` due to https://github.com/Project-OSRM/osrm-backend/pull/3603/files/50b03a93f9472a3e726fc82d6d452ecb1561054f..3bde4426d491540190d403b889f5a8fcf702164a#diff-d79e8547c1b4e3f383ca1879d7db6bebL35 and my compiler complained about the double `const`